### ISSUE TITLE
Please reopen #3244

### ISSUE THREAD

**Issue by picrap** [MahApps/MahApps.Metro#3600]
The issue #3244 is still here and there is no way to reopen it, so I create a new issue for this...

**Comment by punker76** [comment]
@picrap After #3511 and ca9c76ceb1cb82aa904b19fec09c052d728d6573 the default for the left and right window commands behavior is set to `Never`. So if you set `ShowTitleBar` to false they will not shown. You must set this to `HiddenTitleBar` if you want to show these commands. This will be available in v2.0 and you can this behavior with the latest pre-release.

**Comment by RavinduL** [comment]
@punker76 as of `2.4.6`,

```xaml
<mah:MetroWindow
	ShowTitleBar="False"
	RightWindowCommandsOverlayBehavior="HiddenTitleBar"
```

...looks like 👇,

![Actual](https://user-images.githubusercontent.com/7352580/122841831-72f52180-d33f-11eb-800a-49a22d2857cc.png)

Shouldn't look like 👇 instead, with the view extending into the titlebar?

![Expected](https://user-images.githubusercontent.com/7352580/122841890-97e99480-d33f-11eb-879c-a3e282cc7bd5.png)

**Comment by picrap** [comment]
Same problem as @ravindUwU , we have a full title bar that we can’t remove 😭  @punker76

**Comment by BennyBread** [comment]
Same Problem here:
normal:
![image](https://github.com/user-attachments/assets/de72fe52-948e-4796-9b02-365ea441c559)

ShowTitleBar="False"
RightWindowCommandsOverlayBehavior="HiddenTitleBar"
![image](https://github.com/user-attachments/assets/154025a9-dcd6-4aa6-8598-40bd81027f6f)

ShowTitleBar="False"
![image](https://github.com/user-attachments/assets/bba91e15-fa54-4f20-b6d8-3edc74fa6d54)


I don't know which one to prefer.
I like the third most, but the right commands are missing then...
